resources/local/config.toml

```diff
@@ -121,6 +127,9 @@ event_stream_buffer_length = 100
 # The capacity of the broadcast channel size.
 broadcast_channel_size = 100
 
+# The max rate of requests (per second) before they are limited.
+qps_limit = 1000
```

Comment:
is this even relevant to sse?

Reply:
Any http endpoint could be overwhelmed by too many requests. My thinking at the time was to apply the rate limiting across all endpoints, even if they are used for long lived connections like in the case of SSE.